_health-care/_js/_components/_health-care-app.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { Nav } from './_hello-world.jsx';
```

Comment:
Yes. We are "destructing" the import.
  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Conceptually, the result of `from './_hello-world.jsx'` is an object with fields named for each `export` statement in the module. Unless there is a default export, you have to name the field somehow.  Without the braced, you'd actually have to do.

```
import HelloWorldExports from './_hello-world.jsx'; 
const Nav = HelloWorldExports.Nav;
```